framework/src/vectorpostprocessors/VectorPostprocessor.C

```diff
@@ -30,8 +30,21 @@ VectorPostprocessor::validParams()
                         "only a single file is output and updated with each invocation");
 
   // VPPs can set this to true if their resulting vectors are naturally replicated in parallel
-  // setting this to true will keep MOOSE from unnecesarily broadcasting those vectors
-  params.addPrivateParam<bool>("_is_broadcast", false);
+  // setting this to false will keep MOOSE from unnecessarily broadcasting those vectors
+  params.addPrivateParam<bool>("_auto_broadcast", true);
+
+  // VPPs can operate in "distributed" mode, which disables automatic the automatic broadcasting
+  // and results in an individual file per processor if CSV output is enabled
+  MooseEnum parallel_type("DISTRIBUTED REPLICATED", "REPLICATED");
```

Comment:
I don't like these enums. They are actually backwards from the mesh use. On the mesh, replicated means that each processor has a full copy of the mesh, distributed means that each processor owns one piece of the mesh. Here distributed means each processor has a full copy of the vector and replicated means only processor zero has it? 

Maybe consider "gathered" "scattered"? What about flipping the meaning of these enums? We can chat about this a bit if you want.

Reply:
No, "DISTRIBUTED" means that each VPP has a portion of the vector.